lib/recommendation-server.js

```diff
@@ -82,17 +89,36 @@ RecommendationServer.prototype = {
       req.send();
     }
   },
+
+  getResultClass: function(data) {
+    let types = Object.keys(data.enhancements);
+    if (types.indexOf('tld') > -1) {
+      return this.resultClasses.tld;
+    } else if (types.indexOf('wikipedia') > -1) {
+      return this.resultClasses.wikipedia;
+    }
+    return null;
```

Comment:
You don't need to use `Object.keys` here. Just check for tld like this: `if ('tld' in data.enhancements)` and ditto for looking for the wikipedia key in the enhancements object.

There's no need to return null; undefined is returned by default, which works for falsy checks.